src/components/VDialog/VDialog.js

```diff
@@ -81,6 +81,9 @@ export default {
         'v-dialog__content': true,
         'v-dialog__content--active': this.isActive
       }
+    },
+    resetScroll () {
+       document.getElementsByClassName('dialog dialog--active')[0].scrollTop = 0;
```

Comment:
`this.$refs.dialog && this.$refs.dialog.$el.scrollTop = 0`

There are also lint errors (and please run tests to make sure they pass)

Reply:
I was using the class '.dialog-active' to get the element, but this class does not exists before show. Your suggestion (this.$refs.dialog && this.$refs.dialog.$el.scrollTop = 0) is better because does not have this limitation. So, reset scroll at showing using $refs is the best solution.
Thanks for your time.